analyses/tp53_nf1_score/run_classifier.sh

```diff
@@ -56,10 +56,16 @@ Rscript --vanilla ${analysis_dir}/00-tp53-nf1-alterations.R \
   --outputFolder ${analysis_dir}/results \
   --gencode ${cds_file}
 
-# expression files for prediction
-collapsed_stranded="${data_dir}/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds"
-collapsed_polya="${data_dir}/pbta-gene-expression-rsem-fpkm-collapsed.polya.rds"
-
+if [[ RUN_FOR_SUBTYPING == "0" ]]
```

Comment:
Just noting this addition, which makes sense in the context of the run for subtyping script, but does have some (very small) risk of running with outdated data.